src/studio/screens/console.c

```diff
@@ -4270,7 +4270,10 @@ static void processKeyboard(Console* console)
         }
         else
         {
-            if(keyWasPressed(console->studio, tic_key_up)) onHistoryUp(console);
+            if(keyWasPressed(console->studio, tic_key_up)) {
+			    onClsCommand(console);
```

Comment:
hmm, I don't understand why we need to clear console everytime we press UP button, 
I think we need to scroll screen to make text visible instead cleaning

Reply:
In the end, the proper solution was very easy, I'm going to apply it, cheers :)